lib/os.js

```diff
@@ -27,21 +27,26 @@ exports.platform = function() {
 const trailingSlashRe = isWindows ? /[^:]\\$/
                                   : /.\/$/;
 
+var _tmpdir;
```

Comment:
Maybe it's a common pattern I've just not noticed enough, but is this underscore necessary?